    /**
     * This test should fail with an with an IOException because there is
     * actually no such JNDI to connect to, however the datastore is
     * {@code null} so that never happens...
     *
     * @throws IOException
     */
    @Test(expected = IOException.class)
    public void testCanGetJTDSdataStore() throws IOException {
        jndiProps.put("dbtype", "jtds-sqlserver");

        JDBCDataStore dataStore = (JDBCDataStore) DataStoreFinder.getDataStore(jndiProps);
        assertNotNull("The datastore should not be 'null'", dataStore);
        fail("Expected exception was not thrown");
    }

